app/views/dradis/plugins/calculators/cvss/_ce_tools_menu.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if !defined?(Dradis::Pro) %>
+  <li>
+    <%= link_to 'Risk Calculators - CVSS', cvss_calculator.calculators_cvss_path, class: 'dropdown-item', data: { turbolinks: false } %>
+  </li>
+<% end %>
```

Comment:
We don't need this guard since Pro will never be defined in CE, and this will never be called from Pro.
```suggestion

<li>
  <%= link_to 'Risk Calculators - CVSS', cvss_calculator.calculators_cvss_path, class: 'dropdown-item', data: { turbolinks: false } %>
</li>
```